review

```diff
@@ -206,9 +170,9 @@ review() {
     hr
 }
 
-obtainReviewRemote
-obtainReviewBranch
-obtainReviewUrl
-obtainCurrentBranch
+reviewRemote=$(obtainReviewRemote)
```

Comment:
Is it necessary? I though bash has a global scope (sic!) for all variables so you don't need to redefine them second time.

This convention obtainSomeVariable is nice - it is explicitly showing you that you obtain "someVariable" to your global scope.

Reply:
Oh, I didn't think of that. 😕